src/widgets/hero-page/ui/hero-page.module.scss

```diff
@@ -0,0 +1,54 @@
+.hero-page {
+  gap: 16px;
+  min-height: 440px; // TODO ask about design in Figma - 700px there and check @media tablet etc
```

Comment:
What about to place such TODOs to your personal notebook. In my opinion for other devs it's useless

Reply:
Removed